builder.config.json

```diff
@@ -85,105 +85,122 @@
         "type": "ConsoleIO.Print",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 5
       },
       "ConsoleIO.Read": {
         "name": "ConsoleIO.Read",
         "type": "ConsoleIO.Read",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 5
       },
       "Core.Case": {
         "name": "Core.Case",
         "type": "Core.Case",
         "is_interactive": false,
         "is_branching": true,
-        "category":  2
+        "category":  2,
+        "menu_category": 3
       },
       "Core.Log": {
         "name": "Core.Log",
         "type": "Core.Log",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 5
       },
       "Core.Output": {
         "name": "Core.Output",
         "type": "Core.Output",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 1
       },
       "Core.RunFlow": {
         "name": "Core.RunFlow",
         "type": "Core.RunFlow",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 2
       },
       "Core.SetContactProperty": {
         "name": "Core.SetContactProperty",
         "type": "Core.SetContactProperty",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 2
       },
       "Core.SetGroupMembership": {
         "name": "Core.SetGroupMembership",
         "type": "Core.SetGroupMembership",
         "is_interactive": false,
         "is_branching": false,
-        "category":  0
+        "category":  0,
+        "menu_category": 2
       },
       "MobilePrimitives.Message": {
         "name": "MobilePrimitives.Message",
         "type": "MobilePrimitives.Message",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 1
       },
       "MobilePrimitives.NumericResponse": {
         "name": "MobilePrimitives.NumericResponse",
         "type": "MobilePrimitives.NumericResponse",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 1
       },
       "MobilePrimitives.OpenResponse": {
         "name": "MobilePrimitives.OpenResponse",
         "type": "MobilePrimitives.OpenResponse",
         "is_interactive": true,
         "is_branching": false,
-        "category":  1
+        "category":  1,
+        "menu_category": 1,
+        "divider_before": true
```

Comment:
This could be fragile. What do you think of something like having numbers that indicate subcategories instead? 
Eg. 
- All 1's would have no dividers. 
- [{category: 1.1}, {category: 1.1}, {category: 1.2}] would have a divider between 1.1 and 1.2?